documentation/assemblies/assembly-using-the-user-operator.adoc

```diff
@@ -13,6 +13,10 @@ The _Deploying Strimzi_ guide provides instructions to deploy the User Operator:
 * link:{BookURLDeploying}#deploying-the-user-operator-using-the-cluster-operator-{context}[Using the Cluster Operator (recommended)^]
 * link:{BookURLDeploying}#deploying-the-user-operator-standalone-{context}[Standalone to operate with Kafka clusters not managed by Strimzi^]
 
-include::../modules/ref-operator-user.adoc[leveloffset=+1]
-include::../modules/proc-configuring-kafka-user.adoc[leveloffset=+1]
+For more information about the schema, see xref:type-KafkaUser-reference[`KafkaUser` schema reference].
+
+.Authenticating and authorizing access to Kafka
+You configure `KafkaUser` properties to set up authentication and authorization mechanisms to manage client access to Kafka.
```

Comment:
Can we make it clear this this is authentication and authorization mechanisms enabled for this particular user?

Reply:
You configure `KafkaUser` properties to set up authentication and authorization mechanisms that a specific client is allowed to use to access Kafka.

Reply:
> Use `KafkaUser` to enable the authentication and authorization mechanisms that a specific client uses to access Kafka.